content/about/careers/headless/2024-4-software-engineering-manager.md

```diff
@@ -0,0 +1,9 @@
+---
+date: 2024-04-16
+title: Software Engineering Manager
+---
+
+We seek a talented engineering manager capable of translating organizational goals between the executive and engineering teams, supporting, and mentoring technical staff, and providing strong, lightly held opinions for technical guidance.
```

Comment:
That phrase is (IMNSHO) a cliche thats confusing and/or easily mis-interpretable, and its intent should be obvious to anyone with a scientific mind.  Would you accept this instead?
```suggestion
We seek a talented engineering manager capable of translating organizational goals between the executive and engineering teams, supporting, and mentoring technical staff, and providing sage yet flexible opinions for technical guidance.
```

Reply:
👍